.env.local.example

```diff
@@ -1,6 +1,11 @@
 # Allows you to view CMS seed data
 DATOCMS_READ_ONLY_API_TOKEN=a45ce78f9b2053c229bbfe9e3fca7b
 
+# Get these values if you are using Agility CMS (https://manager.agilitycms.com/settings/apikeys)
```

Comment:
To confirm, this is a read-only key, correct?

Reply:
Correct!